lib/build-metadata.ts

```diff
@@ -59,11 +59,14 @@ export class BuildMetadata {
 	public parseMetadata() {
 		// Yaml takes precedence over json (as our docs are in
 		// yaml), but balena takes precedence over resin
+		// .yml vs .yaml: https://stackoverflow.com/questions/21059124/is-it-yaml-or-yml/
 		const potentials = [
 			{ name: 'balena.yml', type: MetadataFileType.Yaml },
+			{ name: 'balena.yaml', type: MetadataFileType.Yaml },
 			{ name: 'balena.json', type: MetadataFileType.Json },
 			{ name: 'resin.yml', type: MetadataFileType.Yaml },
-			{ name: 'balena.json', type: MetadataFileType.Json },
+			{ name: 'resin.yaml', type: MetadataFileType.Yaml },
+			{ name: 'resin.json', type: MetadataFileType.Json },
```

Comment:
Note that the changes above are not just adding 'yaml', but also changing `balena.json` to `resin.json`.